### ISSUE TITLE
Channel 15 weird behaviour when GRPPWM set

### ISSUE THREAD

**Issue by aspyra** [RobTillaart/PCA9635#7]
I am experiencing some problems with channel 15.

When driver mode is set to _PCA9635_LEDOFF_ / _PCA9635_LEDON_, the channel works correctly.
However, if I set each channel to _PCA9635_LEDGRPPWM_, set each channel's PWM to 255, and sweep through group PWM values, all channels work correctly - except the last. There is no output, checked with an oscilloscope.

This is what I do in setup:
```
ledArray.begin();
ledArray.writeMode(PCA9635_MODE2, 0b00010101);
delay(1);
for (int channel = 0; channel < 16; channel++)
{
    ledArray.setLedDriverMode(channel, PCA9635_LEDGRPPWM);
    ledArray.write1(channel, 255);
    delay(1);
}
ledArray.setGroupPWM(0);
```
And this is the sweep:
```
//every 2ms
 if (pwms < 256)
     pwm::ledArray.setGroupPWM(pwms);
else
     pwm::ledArray.setGroupPWM(511 - pwms);

if (++pwms == 512)
     pwms = 0;
```

At first, I thought that I might have damaged the driver while probing, but I assembled another board and it has the same problem. I have checked the PCB design and the physical board, and there is no difference between the circuits of all channels.

**Comment by RobTillaart** [comment]
Thanks for reporting, will dive into this tomorrow.

**Comment by aspyra** [comment]
I think i have found the issue. I have added code to check for errors after every command, and after the line `write1(15, 255)` error _0xFE_ is returned. I have looked into the library code and `write1(15, 255)` is calling `writeN(15, 255, 1)`, which fails on the first check:
`if (channel + count > 15)`
If I change the check to `if (channel + count > 16)`, everything works as intended.
I have made a pull request, #8.
